packages/slate-plugins/src/elements/list/onKeyDownList.ts

```diff
@@ -37,4 +40,9 @@ export const onKeyDownList = (options?: ListOnKeyDownOptions) => (
       moveListItemDown(editor, { list, listItem }, options);
     }
   }
+  getOnHotkeyToggleNodeTypeDefault({
```

Comment:
this won't work because we can't toggle lists with `setNodes` (needs wrap/unwrap)

Reply:
This made it into the PR by mistake, reverted.